src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_InvalidExpression.cs

```diff
@@ -570,5 +570,139 @@ void F(int p /*<bind>*/= M()/*</bind>*/)
+  Instance Receiver: 
+    null
+  Arguments(3):
+      IArgumentOperation (ArgumentKind.Explicit, Matching Parameter: format) (OperationKind.Argument, Type: null) (Syntax: 'format: """"')
+        ILiteralOperation (OperationKind.Literal, Type: System.String, Constant: """") (Syntax: '""""')
+        InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+        OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+      IArgumentOperation (ArgumentKind.Explicit, Matching Parameter: format) (OperationKind.Argument, Type: null, IsInvalid) (Syntax: 'format: """"')
+        ILiteralOperation (OperationKind.Literal, Type: System.String, Constant: """") (Syntax: '""""')
+        InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+        OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+      IArgumentOperation (ArgumentKind.ParamArray, Matching Parameter: args) (OperationKind.Argument, Type: null, IsInvalid, IsImplicit) (Syntax: 'string.Form ... format: """")')
+        IArrayCreationOperation (OperationKind.ArrayCreation, Type: System.Object[], IsInvalid, IsImplicit) (Syntax: 'string.Form ... format: """")')
+          Dimension Sizes(1):
+              ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 0, IsInvalid, IsImplicit) (Syntax: 'string.Form ... format: """")')
+          Initializer: 
+            IArrayInitializerOperation (0 elements) (OperationKind.ArrayInitializer, Type: null, IsInvalid, IsImplicit) (Syntax: 'string.Form ... format: """")')
+              Element Values(0)
+        InConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+        OutConversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+";
+            VerifyOperationTreeAndDiagnosticsForTest<InvocationExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics: new DiagnosticDescription[]
+            {
+                // file.cs(6,45): error CS1740: Named argument 'format' cannot be specified multiple times
+                //         /*<bind>*/string.Format(format: "", format: "")/*</bind>*/;
+                Diagnostic(ErrorCode.ERR_DuplicateNamedArgument, "format").WithArguments("format").WithLocation(6, 45)
+            });
+        }
+
+        [Fact]
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [WorkItem(20050, "https://github.com/dotnet/roslyn/issues/20050")]
+        public void BuildsArgumentsOperationsForDuplicateExplicitArguments_CorrectOrder()
```

Comment:
>CorrectOrder [](start = 75, length = 12)

I am not sure what "CorrectOrder" mean in this scenario

Reply:
Arguments are not ordered. I'll fix it to be more explicit.

---
In reply to: [177267989](https://github.com/dotnet/roslyn/pull/25730#discussion_r177267989) [](ancestors = 177267989)